bokeh/util/token.py

```diff
@@ -103,7 +103,7 @@ def generate_jwt_token(session_id: str,
     Returns:
         str
     """
-    now = calendar.timegm(dt.datetime.now().utctimetuple())
+    now = calendar.timegm(dt.datetime.utcnow().utctimetuple())
```

Comment:
Given this code code is repeated multiple times, it would be good to factor it out under a descriptive name (say `utc_timestamp`) under e.g. `bokeh.util.time`.

Reply:
How about

```
def utc_timestamp() --> int:
    """
    Calculate the current time in seconds since epoch.
    UTC is used so that results are timezone independent.
    """

    return calendar.timegm(dt.datetime.utcnow().utctimetuple())
```
  
If i put this in ```bokeh/util/time.py``` it will be the only function in the file. Is this OK?

Alternatively I could put it in ```bokeh/utils/token.py``` as right now the calculation is only used for token related uses, admittedly that could change in the future.

Are there any style guide rules about where a new function should be added? Is putting at the start or the end of the General API section preferred, or does it not matter

Reply:
Within each section functions are typically in alphabetical order unless there are dependencies that prevent that. General API is intended to be API that any average user might make use of. I don't think this falls in to that category. Dev API is API that advanced users might use, and that the library itself uses, that we do intend to maintain under test. Private API is used by the library and there are no guarantees about compatibility at all (i.e. may change at any time without warning or deprecation). 

Obviously these distinction are somewhat judgment calls and there is probably alot of "General API" that could be re-classified as Dev API. And some Dev API that could become private (but that would require a deprecation and eventual major version bump)